Sources/TestingMacros/Support/Additions/EditorPlaceholderExprSyntaxAdditions.swift

```diff
@@ -0,0 +1,26 @@
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2024 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+//
+
+#if compiler(>=5.11)
```

Comment:
This seems inverted? We want to use the ACL'ed import don't we?

Reply:
We are using ACL'ed imports. There's a difference in behaviour between the Swift 5.10 and Swift 5.11/6 compilers that we're working around here (you'll get various diagnostics if you don't do this while you've got ACL'ed imports enabled.)